src/olympia/editors/models.py

```diff
@@ -1075,6 +1075,21 @@ def get_auto_approved_queue(cls):
             )
         )
 
+    @classmethod
+    def get_content_review_queue(cls):
+        """Return a queryset of Addon objects that have been auto-approved and
+        need content review."""
+        success_verdict = amo.AUTO_APPROVED
+        a_year_ago = datetime.now() - timedelta(days=365)
+        return (
+            Addon.objects.public()
+            .filter(
+                _current_version__autoapprovalsummary__verdict=success_verdict)
+            .filter(
+                Q(addonapprovalscounter__last_content_review=None) |
+                Q(addonapprovalscounter__last_content_review__lt=a_year_ago))
+        )
```

Comment:
more a nitpick than something that needs changing: These filter statements can be combined with something like

```python
.filter(
    Q(_current_version__autoapprovalsummary__verdict=success_verdict) &
    Q(Q(addonapprovalscounter__last_content_review=None) |
      Q(addonapprovalscounter__last_content_review__lt=a_year_ago)))

```

which… now that I wrote it isn't much nicer. But maybe it's more explicit, anyway… nitpick

Reply:
I almost made the same suggestion but yeah, not sure if it's better either.

Reply:
yeah, I find that version way harder to read :p